Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -771,6 +771,8 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 // Defaults to YES; if set to NO, WebKit will draw the grey wash and highlights itself.
 @property (nonatomic, setter=_setUsePlatformFindUI:) BOOL _usePlatformFindUI WK_API_AVAILABLE(macos(10.15));
 
+@property (nonatomic, readonly, getter=_isDisplayingPDF) BOOL _displayingPDF WK_API_AVAILABLE(macos(WK_MAC_TBA));
```

Comment:
Shouldn't this be the iOS version but moved to a shared place???

Reply:
Well, yes, but the two impls are distinct, so now we have the property `_displayingPDF` in both the `WKPrivateMac` and the `WKPrivateIOS` categories, with the former implemented in `WKWebViewMac` and the latter implemented in `WKWebViewIOS`.

Reply:
In hindsight, this fragmented declaration is markedly worse than moving to a shared place, so I'll just do that instead.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/030e8c9ccab5aff0053a0ba85d32d9ffdeb58b9a